content/applications/sales/point_of_sale/overview/customer_notes.rst

```diff
@@ -0,0 +1,38 @@
+==============
+Customer notes
+==============
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
```

Comment:
```suggestion
For instance, to provide cleaning and maintenance tips; they can also be used to track a
```

Reply:
I've added the first part, but left it as 2 sentences (with ".")